app/models/grda_warehouse_base.rb

```diff
@@ -55,4 +55,16 @@ def paper_trail_meta_value(key)
       value
     end
   end
+
+  def self.references_hud_client?
+    columns_hash.key?('PersonalID') && reflect_on_association(:client)
```

Comment:
`?` convention should return a boolean (I think) but this returns the reflection or `false` or `nil`

Reply:
good thought, the type could be tighter, will fix